README.md

```diff
@@ -59,24 +59,27 @@ Looking for example to use `react-flexbox-grid`? Head over to [react-flexbox-gri
 
 Advanced composition
 -------
-Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you have a `MyFormInput` component that should also act as both a `Row` and a `Col`.
+Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you're using a third party component that accepts a className.
 
 ```js
-import {getRowClassNames, getColClassNames} from 'react-flexbox-grid'
+// a component that accepts a className
+import RandomComponent from 'some-package';
```

Comment:
Yes, `RandomComponent` isn't a super great name.  I've updated that and (hopefully) improved  the explanation a bit.

I'm also wondering if it'd make sense to export the `PropTypes` for the Column and Row?  That would allow the `MyComponent` example above to validate it's passed the correct props without having to re-create the list.